src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    void drawFigure();
```

Comment:
The method name `drawFigure` could be simplified to `draw` as it is already within the `Drawable` interface, which implies that the action is related to drawing. Method names should be as simple and as descriptive as possible.

Reply:
```suggestion
    void draw();
```